src/quo/components/list_items/saved_address/view.cljs

```diff
@@ -8,19 +8,20 @@
     [quo.foundations.colors :as colors]
     [quo.theme :as quo.theme]
     [react-native.core :as rn]
-    [reagent.core :as reagent]))
+    [reagent.core :as reagent]
+    [utils.address :as address]))
 
 (defn- left-container
-  [{:keys [blur? theme name address customization-color]}]
+  [{:keys [blur? theme name ens address customization-color]}]
   (let [names      (remove string/blank? (string/split name " "))
         first-name (if (> (count names) 0) (first names) "")
         last-name  (if (> (count names) 1) (last names) "")]
     [rn/view {:style style/left-container}
      [wallet-user-avatar/wallet-user-avatar
-      {:size   :medium
-       :f-name first-name
-       :l-name last-name
-       :color  customization-color}]
+      {:size                :medium
+       :f-name              first-name
```

Comment:
the api for wallet user avatar has adjusted recently, is this correct now? 🤔

Reply:
Adjusted